conf/modules.config

```diff
@@ -1126,37 +1140,68 @@ if (!params.skip_multiqc) {
 }
 
 //
-// Salmon pseudo-alignment options
+// Salmon/ Kallisto pseudo-alignment options
 //
 
-if (!params.skip_pseudo_alignment && params.pseudo_aligner == 'salmon') {
+if (params.pseudo_aligner == 'salmon') {
     process {
-        withName: '.*:QUANTIFY_SALMON:SALMON_QUANT' {
+        withName: '.*:QUANTIFY_PSEUDO:SALMON_QUANT' {
```

Comment:
```suggestion
        withName: '.*:QUANTIFY_PSEUDO_ALIGNMENT:SALMON_QUANT' {
```

Reply:
Might be better to be explicit with the name of the subworkflow like we are everywhere else.